src/Raven.Server/Rachis/Commands/Leader.RachisMergedCommand.cs

```diff
@@ -155,6 +162,14 @@ private void InsertCommandToLeaderLog(ClusterOperationContext context)
 
             }
 
+            private void ValidateTerm(long term)
```

Comment:
let's have a single method

Reply:
Cant do it because it throws a different exception

Reply:
so? also, let's make it `TermValidationException`